scripts/world_bank/datasets/datasets.py

```diff
@@ -351,7 +378,8 @@ def write_wb_codes():
 def get_all_codes():
     all_codes = {}
     for file_name in os.listdir(DOWNLOADS_DIR):
-        if file_name.endswith(CSV_ZIP_FILE_SUFFIX):
+        if file_name.endswith(CSV_ZIP_FILE_SUFFIX) or file_name.endswith(
```

Comment:
if case is a an issue, can do
`if file_name.lower().endswith(CSV_ZIP_FILE_SUFFIX.lowe()):`